hmda/src/main/scala/hmda/validation/rules/lar/validity/2021/V695_1.scala

```diff
@@ -12,8 +13,15 @@ object V695_1 extends EditCheck[LoanApplicationRegister] {
   override def parent: String = "V695"
 
   override def apply(lar: LoanApplicationRegister): ValidationResult = {
-    (lar.larIdentifier.NMLSRIdentifier is integer) or 
-    (lar.larIdentifier.NMLSRIdentifier is oneOf("Exempt", "NA"))
+
+    val nmlsrID = lar.larIdentifier.NMLSRIdentifier
+
+    (nmlsrID is integer) or
+    (nmlsrID is oneOf("Exempt", "NA")) or
+      (
+        (toValidBigInt(nmlsrID) is greaterThan(0)) and
```

Comment:
Should this change be made in a 2025 year file version instead of the 2021 file?

Reply:
Negative,  since NMLS ID can be up to 12 digits this has the potential to retroactively fail on past submissions. 

And this integer check is insufficient for such a value as anything past 10 digits fails max int. So it has to be big int.

@Kibrael  what are you thoughts on this? lol